<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuGraph</class>
 <widget class="QDialog" name="MenuGraph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="background_graph">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>154</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="layout_menu">
    <item>
     <widget class="QCommandLinkButton" name="btn_sys">
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCommandLinkButton" name="btn_graph">
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCommandLinkButton" name="btn_bench">
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCommandLinkButton" name="btn_net">
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_ram">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>121</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>RAM utilization:</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>89</x>
     <y>40</y>
     <width>701</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="layout_ram"/>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>350</y>
     <width>131</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Disk organization :</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="brow_disk">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>340</y>
     <width>431</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
